.github/workflows/unittests.yml

```diff
@@ -61,13 +61,19 @@ jobs:
         run: |
           curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/bazelisk-darwin-amd64 > bazelisk
           chmod +x bazelisk
+      - name: Install GNU find
+        run: |
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install findutils
+          ln -sf /usr/local/bin/gfind /usr/local/bin/find
```

Comment:
On macOS we need to install GNU find to support the syntax used [here](https://github.com/larq/compute-engine/pull/248/files#diff-21587c1c7cea11803fce4eb498c90aa5R40).